pkg/proxy/integrations/genericParser/util.go

```diff
@@ -4,24 +4,23 @@ import (
 	"encoding/base64"
 	"fmt"
 
-	// "fmt"
 	"unicode"
 
 	"go.keploy.io/server/pkg/hooks"
 	"go.keploy.io/server/pkg/models"
 	"go.keploy.io/server/pkg/proxy/util"
+	"go.uber.org/zap"
 )
 
-func PostgresDecoder(encoded string) ([]byte, error) {
+func PostgresDecoder(logger *zap.Logger, encoded string) []byte {
 	// decode the base 64 encoded string to buffer ..
 
 	data, err := base64.StdEncoding.DecodeString(encoded)
 	if err != nil {
-		// fmt.Println(hooks.Emoji+"failed to decode the data", err)
-		return nil, err
+		logger.Error(err.Error())
```

Comment:
Please log the complete error like `failed to decode the encoded byte` or something this which will give more context.